packages/svelte-vscode/package.json

```diff
@@ -9,7 +9,7 @@
         "build": "npm run build:ts && npm run build:grammar",
         "vscode:prepublish": "npm run build && npm prune --production",
         "watch": "npm run build:grammar && tsc -w -p ./",
-        "test": "echo 'NOOP'"
+        "test": "npm run build:grammar && node test/grammar/test.js"
```

Comment:
The reason for this `test.js` script is that we can just read the `dummy` directory to build the argument list instead of having to write a big long list in the `package.json`